### PR TITLE
MSR: get rid of unnecessary templates

### PR DESCRIPTION
**Pull request by Miepee**

### CODE REVIEW

File: randovania/games/samus_returns/logic_database/Area 2 Dam Interior.txt [hunk 1]
```diff
@@ -281,7 +281,7 @@ Extra - asset_id: collision_camera011
   > Out of Bounds in Tunnel
       All of the following:
           # https://youtu.be/f0sq6fDBCjU
-          Out of Bounds Movement (Advanced) and Phase Drift Clip (Advanced) and Can Phase Drift Clip and Shoot Any Missile and Shoot Beam Burst
+          Phase Drift and Out of Bounds Movement (Advanced) and Phase Drift Clip (Advanced) and Shoot Any Missile and Shoot Beam Burst
```
Comment (dyceron) [inline randovania/games/samus_returns/logic_database/Area 2 Dam Interior.txt:284]: Need morph

File: randovania/games/samus_returns/logic_database/header.txt [hunk 2]
```diff
@@ -322,10 +322,6 @@ Templates
           # Spin Jump over the Gravitt when it lunges
           Dodge Enemies (Intermediate)
 
-* Can Phase Drift Clip:
-      # TODO: remove this after the other todos
-      Morph Ball and Phase Drift
-
 * Defeat Swarms:
```
Comment (dyceron) [inline randovania/games/samus_returns/logic_database/header.txt:325]: Is this being removed?

**Review by dyceron — CHANGES_REQUESTED** [review]

File: randovania/games/samus_returns/logic_database/header.txt (hunk 2 above)
Comment (Miepee) [reply randovania/games/samus_returns/logic_database/header.txt:325]: yes

File: randovania/games/samus_returns/logic_database/Area 2 Dam Interior.txt (hunk 1 above)
Comment (Miepee) [reply randovania/games/samus_returns/logic_database/Area 2 Dam Interior.txt]: thx

**Review by dyceron — APPROVED** [review]

**Pull request merged**